cfgov/housing_counselor/jinja2/housing_counselor/improved_index.html

```diff
@@ -180,42 +177,52 @@ <h2 class="h4">
-                                            {% if counselor.adr2 and counselor.adr2 != ' '  %}
-                                                <br>{{ counselor.adr2 }}
-                                            {% endif %}<br>
-                                            {{ counselor.city }}, {{ counselor.statecd }} {{ counselor.zipcd }}
+                                                <a class="a-link a-link__icon" href="{{ counselor.weburl }}">
+                                                  <span class="a-link_text">{{ counselor.nme }}</span>
+                                                  {{ svg_icon( 'external-link' ) }}
+                                                </a>
+                                                <br>
+                                                {{ counselor.adr1 }}
+                                                {% if counselor.adr2 and counselor.adr2 != ' '  %}
+                                                <br>
+                                                {{ counselor.adr2 }}
+                                                {% endif %}
+                                                <br>
+                                                {{ counselor.city }}, {{ counselor.statecd }} {{ counselor.zipcd }}
                                             </p>
-                                            <p><b>Website:</b>
+                                            <p>
+                                                <b>Website:</b>
                                                 <a class="a-link a-link__icon" href="{{ counselor.weburl }}">
-                                                    <span class="a-link_text">{{ counselor.weburl }}</span> {{ svg_icon( 'external-link' ) }}
-                                                </a><br>
-                                                <b>Phone:</b> {{ counselor.phone1 }}<br>
-                                                <b>Email Address:</b> {{ counselor.email }}<br>
+                                                    <span class="a-link_text">{{ counselor.weburl }}</span>
+                                                    {{ svg_icon( 'external-link' ) }}
+                                                </a>
+                                                <br>
+                                                <b>Phone:</b> {{ counselor.phone1 }}
+                                                <br>
+                                                <b>Email Address:</b> {{ counselor.email }}
+                                                <br>
                                                 <b>Languages:</b> {{ counselor.languages | join( ', ' ) }}
                                             </p>
                                         </div>
                                     </td>
-                                    <td data-label="Services">
+                                    <td>
```

Comment:
I think the `data-label` needs to stay there so the label will appear when the table columns stack on top of each other on small screens

Reply:
Fixed!